src/DrawFiveTest.java

```diff
@@ -0,0 +1,27 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+public class DrawFiveTest {
+    private UnoFlipModel unoFlipModel;
+
+    private Card testCard;
+
+    @Before
+    public void setup(){
+        unoFlipModel = new UnoFlipModel();
+        unoFlipModel.getPlayers().add(new Player("test1"));
+        unoFlipModel.getPlayers().add(new Player("test2"));
+        unoFlipModel.setNumPlayers(2);
+        testCard = new Card(Card.Rank.DRAW_ONE, Card.Colour.RED,Card.Rank.DRAW_FIVE, Card.Colour.ORANGE); //false since card is on darkside
```

Comment:
what did you mean by this comment?

Reply:
forgot to take it out ( was old comment i made)